README.mkd

```diff
@@ -296,16 +296,27 @@ in the ratified base ISA and extensions (e.g. the use of 'w', 'd',
 
 Vendor                 | Prefix          | URL
 :--------------------- | :-------------- | :-------------
+Open Hardware Group    | cv              | https://www.openhwgroup.org/
 SiFive                 | sf              | https://www.sifive.com/
 T-Head                 | th              | https://www.t-head.cn/
 Ventana Micro Systems  | vt              | https://www.ventanamicro.com/
 
 NOTE: Vendor prefixes are case-insensitive.
 
+NOTE: OpenHW cores are all branded as CORE-V, hence the prefix.
+
 ### List of vendor extensions
 
 Vendor  | Name            | Version        | ISA Document
 :------ | :-------------- | :------------- | :---------------
+OpenHW  | Xcvalu          | 2.0.0          | [CORE-V Instruction Set Extensions](https://docs.openhwgroup.org/projects/cv32e40p-user-manual/en/latest/instruction_set_extensions.html)
```

Comment:
I could not find any reference to `Xcvalu` or the other extension names on the linked page (even when searching for these extension names using the search box on the top left).

Also, I cannot see how the specified version `2.0.0` correlates to the contents of the linked documentation.

Reply:
@cmuellner Ah - I linked to the main architecture description of the architectural extensions. There are separate documents  describing the `-march` naming convention and listing the builtin functions for those extensions.  Rather than make the table very large, should I provide those links generically for CORE-V at the bottom of the table? Thanks.

Reply:
A link to a document where the extensions are defined (name, version and included instructions) would be sufficient. I.e. the extension name `xcvalu` and its version should not be defined in the toolchain conventions but in the (referenced) extension documentation.